### PR TITLE
update isis feature.textproto

### PR DESCRIPTION
**Pull request by dplore**
* Fixed large number of path errors due to model changes 
* Fixed paths with typo in interfaces/interface
* added folder for link-state-database as there are quite a lot of lsdb telemetry paths and seemed good to separate for a separate test
* I did not include two paths which were thought to be required but are no longer part of the OC model set:
`/network-instances/network-instance/protocols/protocol/isis/interfaces/interface/levels/level/adjacencies/adjacency/state/local-extended-system-id`
`/network-instances/network-instance/protocols/protocol/isis/interfaces/interface/levels/level/adjacencies/adjacency/state/remaining-hold-time`

### CODE REVIEW

**Comment by dplore** [comment]
@bensons after many hours fixing paths, this is ready for review

File: feature/isis/link-state-database/feature.textproto [hunk 1]
```diff
@@ -0,0 +1,156 @@
+id {
+  name: "isis_link-state-database"
```
Comment (robshakir) [inline feature/isis/link-state-database/feature.textproto:2]:
```suggestion
  name: "isis_lsdb"
```

**Review by robshakir — APPROVED** [review]

**Review by bensons — APPROVED** [review]

File: feature/isis/link-state-database/feature.textproto (hunk 1 above)
Comment (bensons) [reply feature/isis/link-state-database/feature.textproto:2]: It seems to me that we should keep the name of the Feature Profile consistent with the directory structure, when possible... I think there are some tools that make this assumption.

Comment (dplore) [reply feature/isis/link-state-database/feature.textproto:2]: concise and readable suggestion, but the OC path spells this out, so I wanted to align with that for consistency

**Pull request merged**